python/core/auto_generated/pointcloud/qgspointcloudlayer.sip.in

```diff
@@ -50,6 +50,7 @@ Constructor for LayerOptions with optional ``transformContext``.
 
     explicit QgsPointCloudLayer( const QString &path = QString(),
                                  const QString &baseName = QString(),
+                                 const QString &dataSourceType = QString(),
```

Comment:
This is an api break. But can't we avoid the whole "dataSourceType" argument by checking if the path starts with http:/https:// instead? It's certainly much cleaner

Reply:
That's a really good idea, I will change to using this. Thanks